### ISSUE TITLE
Dev Server doesn't show linter errors in browser

### ISSUE THREAD

**Issue by Arterixs** [ModyQyW/vite-plugin-eslint2#42]
### Describe the bug

When using nuxt/eslint, linter errors are displayed in the console but not in the browser. A similar issue was mentioned in the nuxt/eslint repository "[Request for support on failOnError Overlay when applying nuxt/eslint](https://github.com/nuxt/eslint/issues/460)"

My package.json: 
```js
"devDependencies": {
    "@nuxt/eslint": "^0.7.5",
    "@types/eslint-config-prettier": "^6.11.3",
    "eslint": "^9.18.0",
    "eslint-config-prettier": "^10.0.1",
    "eslint-plugin-prettier": "^5.2.3",
    "eslint-plugin-unused-imports": "^4.1.4",
    "prettier": "3.4.2",
    "typescript": "^5.7.3",
    "vite-plugin-eslint2": "^5.0.3",
    "vue-tsc": "^2.1.10"
  }
```

My vite.config

```
import eslint from 'vite-plugin-eslint2';

export default defineNuxtConfig({
  ssr: false,

  modules: [ '@nuxt/eslint'],

  css: ['~/assets/css/main.css'],

  typescript: {
     typeCheck: true
   },

  eslint: {
    checker: true,
    config: {
        typescript: true
     }
   },

  vite: {
    plugins: [eslint()]
  }
});
```

Are there any suggestions to solve this problem?

### Reproduction

No the related parameter.

### System Info

```Shell
No the related parameter.
```

### Used Package Manager

npm

### Validations

- [x] Follow our [Code of Conduct](https://github.com/uni-helper/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guide](https://github.com/antfu/contribute) and [Notes from a tired maintainer](https://github.com/pi0/tired-maintainer).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [x] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

### Contributions

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests (actually just go ahead and do it, thanks!)

**Comment by ModyQyW** [comment]
It seems that @nuxt/eslint is using lintInWorker https://github.com/nuxt/eslint/blob/eb9d5fa47f74d68e2cb75b161d4d209cfdd51a45/packages/module/src/modules/checker.ts#L65.

It is documented [here](https://vite-plugin-eslint2.modyqyw.top/guide/options.html#lintinworker)

> When lint in worker, Vite build process will be faster. You will not see Vite error overlay, Vite build process will not be stopped, even with errors shown in terminal.

So, it is a expected behavior. I assume that you are looking for a vite-plugin-checker experience? Lint in worker, and overlay in browser?

**Comment by Arterixs** [comment]
That would be great, thank you)

**Comment by ModyQyW** [comment]
I'll try to implement this in my free time, but probably not too soon. Please be patient.
